polaris-tokens/src/themes/base/color.ts

```diff
@@ -400,1106 +235,631 @@ export const color: {
-    value: colorsExperimental.red[11],
-  },
-  'color-border-critical-hover': {
-    value: colorsExperimental.red[10],
+    value: colors.green[5],
   },
   'color-border-critical': {
-    value: colorsExperimental.red[9],
-  },
-  'color-border-critical-subdued': {
-    value: colorsExperimental.red[9],
+    value: colors.red[8],
   },
   'color-border-caution': {
-    value: colorsExperimental.yellow[5],
-  },
-  'color-border-caution-subdued': {
-    value: colorsExperimental.yellow[5],
+    value: colors.yellow[5],
   },
   'color-border-info': {
-    value: colorsExperimental.azure[9],
-  },
-  'color-border-info-subdued': {
-    value: colorsExperimental.azure[9],
-  },
-  'color-border-interactive-active': {
-    value: colorsExperimental.blue[15],
-  },
-  'color-border-interactive-hover': {
-    value: colorsExperimental.blue[14],
-  },
-  'color-border-interactive': {
-    value: colorsExperimental.blue[13],
-  },
-  'color-border-interactive-focus': {
-    value: colorsExperimental.blue[13],
-  },
-  'color-border-interactive-subdued': {
-    value: colorsExperimental.blue[13],
-  },
-  'color-border-magic-strong': {
-    value: colorsExperimental.purple[12],
+    value: colors.azure[8],
   },
   'color-border-magic': {
-    value: colorsExperimental.purple[10],
+    value: colors.purple[10],
   },
   'color-icon-hover': {
-    value: colorsExperimental.gray[15],
+    value: colors.gray[15],
   },
   'color-icon': {
-    value: colorsExperimental.gray[14],
-  },
-  'color-icon-subdued': {
-    value: colorsExperimental.gray[12],
+    value: colors.gray[14],
   },
   'color-icon-disabled': {
-    value: colorsExperimental.gray[10],
-  },
-  'color-icon-interactive-disabled': {
-    value: colorsExperimental.gray[10],
+    value: colors.gray[10],
   },
   'color-icon-inverse': {
-    value: colorsExperimental.gray[8],
-  },
-  'color-icon-on-color': {
-    value: colorsExperimental.gray[1],
-  },
-  'color-icon-primary': {
-    value: colorsExperimental.gray[16],
+    value: colors.gray[8],
   },
   'color-icon-success': {
-    value: colorsExperimental.green[12],
+    value: colors.green[12],
   },
   'color-icon-critical': {
-    value: colorsExperimental.red[11],
+    value: colors.red[11],
   },
   'color-icon-caution': {
-    value: colorsExperimental.yellow[11],
+    value: colors.yellow[11],
   },
   'color-icon-info': {
-    value: colorsExperimental.azure[11],
+    value: colors.azure[11],
   },
   'color-icon-warning': {
-    value: colorsExperimental.orange[11],
-  },
-  'color-icon-interactive-active': {
-    value: colorsExperimental.blue[15],
-  },
-  'color-icon-interactive-hover': {
-    value: colorsExperimental.blue[14],
-  },
-  'color-icon-interactive': {
-    value: colorsExperimental.blue[13],
-  },
-  'color-icon-interactive-inverse': {
-    value: colorsExperimental.blue[8],
+    value: colors.orange[11],
   },
   'color-icon-magic': {
-    value: colorsExperimental.purple[13],
+    value: colors.purple[13],
```

Comment:
```suggestion
    value: colors.purple[12],
```
This is listed as [12 in Figma](https://www.figma.com/file/oKq2CLPYeX5qK5frzwipM3/Polaris-Gen-3-Styles?type=design&node-id=14696%3A1737&mode=design&t=vGyfRzAa3mogdM3j-1). We need to determine which of these values is correct.

<img width="358" alt="Screenshot 2023-10-02 at 11 56 22 AM" src="https://github.com/Shopify/polaris/assets/21976492/3e190476-a243-4eb6-a4ca-a05cce2e94d2">